packages/massa-web3/src/experimental/basicElements/internal.ts

```diff
@@ -1,5 +1,6 @@
 import Serializer from '../crypto/interfaces/serializer'
 import { Versioner, Version } from '../crypto/interfaces/versioner'
+import { FIRST, ONE } from '../utils'
```

Comment:
If One I would prefer ZERO instead of First. When seing this I thought "First what".

Reply:
`FIRST` is first element of an array or a string.